src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,122 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.exception.DataProcessingException;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String query = "INSERT INTO drivers (name, license_number) "
+                + "VALUES (?, ?)";
+        try {
+            Connection connection = ConnectionUtil.getConnection();
+            PreparedStatement createManufacturerStatement
+                    = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
+            createManufacturerStatement.setString(1, driver.getName());
+            createManufacturerStatement.setString(2, driver.getLicenseNumber());
+            createManufacturerStatement.executeUpdate();
+            ResultSet resultSet = createManufacturerStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                driver.setId(resultSet.getLong(1));
+            }
+            return driver;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't create driver. " + driver + " ",
+                    throwable);
```

Comment:
You can get rid of empty space after driver -> 
throw new DataProcessingException("Couldn't create driver. " + driver, throwable);
Same for the rest